# Sends object to move robot
string obj

---
bool confirmation